### PR TITLE
[Perf] Add summarized test results for performances test on Windows && performance result empty check

### PR DESCRIPTION
**Pull request by AC-Phoenix**
## Pull Request Checklist

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Code compiles correctly with all tests are passed.
- [ ] I've read the [contributing guide](https://github.com/microsoft/HydraLab/blob/main/CONTRIBUTING.md#making-changes-to-the-code) and followed the recommended practices.
- [ ] [Wikis](https://github.com/microsoft/HydraLab/wiki) or [README](https://github.com/microsoft/HydraLab/blob/main/README.md) have been reviewed and added / updated if needed (for bug fixes / features)

### Does this introduce a breaking change?
*If this introduces a breaking change for Hydra Lab users, please describe the impact and migration path.*

- [ ] Yes
- [ ] No

## Pull Request Description

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Build related changes
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Code style update (formatting, renaming) or Documentation content changes
- [ ] Other (please describe): 

**A few words to explain your changes**:

Issue ID: #

### How you tested it
*Please make sure the change is tested, you can test it by adding UTs, do local test and share the screenshots, etc.*

### Feature UI screenshots or Technical design diagrams
*If this is a relatively large or complex change, kick it off by drawing the tech design with PlantUML and explaining why you chose the solution you did and what alternatives you considered, etc...*

### CODE REVIEW

File: common/src/main/java/com/microsoft/hydralab/performance/parsers/WindowsMemoryResultParser.java [hunk 1]
```diff
@@ -53,6 +59,15 @@ public PerformanceTestResult parse(PerformanceTestResult performanceTestResult)
 
                         parsedData.getProcessIdProcessNameMap().put(processId, processName);
                         parsedData.getProcessIdWindowsMemoryMetricsMap().put(processId, windowsMemoryMetrics);
+
+                        averagedData.getProcessIdProcessNameMap().putIfAbsent(processId, processName);
+                        averagedData.getProcessIdWindowsMemoryMetricsMap().putIfAbsent(processId,
+                                new WindowsMemoryParsedData.WindowsMemoryMetrics());
+                        averagedData.getProcessIdWindowsMemoryMetricsMap().get(processId).accumulate(
```
Comment (taoran6) [inline common/src/main/java/com/microsoft/hydralab/performance/parsers/WindowsMemoryResultParser.java:66]: If we accumulate all the memory data, will it cause overflow? I suggest we calculate the average data simultaneously.

File: react/src/component/PerfTestDashboard.jsx [hunk 2]
```diff
@@ -220,17 +230,17 @@ export default class PerfTestDashboard extends React.Component {
...
                 {renderAndroidBatteryChart}
             </div>}
-            {androidMemoryInfo && <div>
+            {!isAndroidMemoryInfoEmpty && <div>
                 <h3> Android Memory report</h3>
                 {androidMemoryMultiSelect}
                 {renderAndroidMemoryChart}
             </div>}
-            {windowsMemoryInfo && <div>
+            {!isWindowsMemoryInfoEmpty && <div>
```
Comment (taoran6) [inline react/src/component/PerfTestDashboard.jsx:243]: If the `InfoEmpty` is `true`, can we replace the chart with a placeholder and indicate that `There is something wrong when parse the memory data, please check the request param or agent`? We can also ask our team to optimize the placeholder :)

**Comment by taoran6** [comment]
I found a bug of the Rechars that the dots styles are inconsistent, could you pls add `dot={false}` to the `<Line />` tag? Thx

**Comment by taoran6** [comment]
I found a bug of the Rechars that the dots styles are inconsistent, could you pls add `dot={false}` to the `<Line />` tag? Thx

image

File: react/src/component/PerfTestDashboard.jsx (hunk 2 above)
Comment (AC-Phoenix) [reply react/src/component/PerfTestDashboard.jsx]: Thanks, updated.

File: common/src/main/java/com/microsoft/hydralab/performance/parsers/WindowsMemoryResultParser.java (hunk 1 above)
Comment (AC-Phoenix) [reply common/src/main/java/com/microsoft/hydralab/performance/parsers/WindowsMemoryResultParser.java]: Agreed, especially for those tests which run for very long periods of time.
Thanks. Updated.

**Review by taoran6 — APPROVED** [review]
